JS/Prototypes/inheritanceWithPrototype.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<body>
+       
+<p>Example of inheritance with the prototype property </p>
+<p>This prototype property is not enumerable; that is, it isn’t accessible in a for/in loop</p>
+
+<script>
```

Comment:
Always create a new file.
Move the below code to a new file and link it on this page.